libinit.c

```diff
@@ -168,7 +168,11 @@ static int MUK_Alkaa(int * argc, char *** argv, int requested, int * provided)
     wrapname = pathname;
 #endif
 
-    void * wrap_so_handle = dlopen(wrapname, RTLD_LAZY | RTLD_DEEPBIND);
+    int flags = RTLD_LAZY;
```

Comment:
For the sake of explicitness and completeness, maybe we should go for:
```suggestion
    int flags = RTLD_LOCAL | RTLD_LAZY;
```
Could you please test if things still work with this change?

Reply:
Yes, this make sense for me. I tested it and it works. :+1: